src/components/Modals/PublishCollectionModal/PublishCollectionModal.tsx

```diff
@@ -1,257 +1,10 @@
-import * as React from 'react'
-import { Network } from '@dcl/schemas'
-import { env } from 'decentraland-commons'
-import { ModalNavigation, Button, Mana, Loader, Field, InputOnChangeData, Form } from 'decentraland-ui'
-import Modal from 'decentraland-dapps/dist/containers/Modal'
-import { t, T } from 'decentraland-dapps/dist/modules/translation/utils'
-import { ItemRarity } from 'modules/item/types'
-import { getBackgroundStyle } from 'modules/item/utils'
-import { emailRegex } from 'lib/validators'
-import { Props, State } from './PublishCollectionModal.types'
-import './PublishCollectionModal.css'
+import React from 'react'
+import { Props } from './PublishCollectionModal.types'
+import PublishCollectionModalOld from '../PublishCollectionModalOld'
+import PublishCollectionModalWithOracle from '../PublishCollectionModalWithOracle'
```

Comment:
I don't know what's going on with Github, but I'm not seeing this file in this PR.

Reply:
https://github.com/decentraland/builder/issues/2044 The issue was already created but I added and extra comment for better context

Reply:
The whole file is just:
```
import React from 'react'
import { Props } from './PublishCollectionModal.types'
import PublishCollectionModalOld from '../PublishCollectionModalOld'
import PublishCollectionModalWithOracle from '../PublishCollectionModalWithOracle'

const PublishCollectionModal = ({ isRaritiesWithOracleEnabled, ...rest }: Props) => {
  return isRaritiesWithOracleEnabled ? <PublishCollectionModalWithOracle {...rest} /> : <PublishCollectionModalOld {...rest} />
}

export default React.memo(PublishCollectionModal)

```